main.tf

```diff
@@ -67,15 +25,9 @@ module "container_definition" {
 
   environment = merge({
     # General Settings
-    EXTERNAL_PORT_HTTPS = var.external_port
-    INTERNAL_PORT_HTTPS = var.internal_port
-    EXTERNAL_PORT_HTTP  = var.external_port
```

Comment:
shouldnt the internal ports stick around? i thought only external load balancer is removed